docs/guides/getting-started-orchestrate-microservices.md

```diff
@@ -48,9 +48,9 @@ Start by designing your automated process using BPMN. This guide introduces you
 
 ### Create credentials for your Zeebe client
 
-To interact with your Camunda Cloud cluster, you'll use the Zeebe client. First, you'll need to create credentials.
+To interact with your Camunda Platform 8 cluster, you'll use the Zeebe client. First, you'll need to create credentials.
 
-1. The main page for Camunda Cloud Console should be open on another tab. Use Camunda Cloud Console to navigate to your clusters either through the navigation **Clusters** or by using the section under **View all** on the **Clusters** section of the main dashboard. Click on your existing cluster. This will open the **Overview** for your cluster, where you can find your cluster id and region. You will need this information later when creating a worker in the next section.
+1. The main page for Camunda Platform 8 Console should be open on another tab. Use Camunda Platform 8 Console to navigate to your clusters either through the navigation **Clusters** or by using the section under **View all** on the **Clusters** section of the main dashboard. Click on your existing cluster. This will open the **Overview** for your cluster, where you can find your cluster id and region. You will need this information later when creating a worker in the next section.
```

Comment:
maybe we can even leave out Camunda Platform 8 here and just say Console? At least the second time we mention it

Reply:
I'm good with this, I'll make the update.